TrkFilters/src/HelixFilter_module.cc

```diff
@@ -191,6 +201,61 @@ namespace mu2e
     }
     return true;
   }
+
+
+  float   HelixFilter::helixHitRatio(const HelixSeed* helix){
+
+    int      nCH = helix->hits().size();
+    int      nFaces(36); //was 36, now doing stations instead of faces
```

Comment:
This should come from StrawId.  The name is also misleading, 36 is # of planes not faces or stations.

Reply:
Thank you for noticing it. I changed it as suggested.